### PR TITLE
chore: add minutes for 2024-09-19 meeting

### PR DESCRIPTION
**Pull request by marco-ippolito**
Fixes: https://github.com/nodejs/Release/issues/1033

@nodejs/releasers

### CODE REVIEW

File: doc/meetings/2024-09-19.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Node.js  Release WorkGroup Meeting 2024-09-19
+
+## Links
+
+* **Recording**:
```
Comment (RafaelGSS) [inline doc/meetings/2024-09-19.md:5]: Missing recording

File: doc/meetings/2024-09-19.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* **GitHub Issue**: <https://github.com/nodejs/Release/issues/1033>
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1xkoISCtODqo5WBcDQ2zqkWim-mUj2IE0zTQpecbVysE>
+
+## Present
+
+* Release team: @nodejs/release
+* LTS team: @nodejs/lts
+* Marco Ippolito
+* Antoine du Hamel (@aduh95)
+* Rafael Gonzaga
```
Comment (RafaelGSS) [inline doc/meetings/2024-09-19.md:15]:
```suggestion
* Rafael Gonzaga (@RafaelGSS)
```

File: doc/meetings/2024-09-19.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+## Agenda
+
+## Announcements
+
+*Extracted from **Release-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/Release
+
+* Current status of LTS group/ backporters [#1012](https://github.com/nodejs/Release/issues/1012)
+ Marco will proceed to merge LTS group to release and close the issue.
```
Comment (richardlau) [inline doc/meetings/2024-09-19.md:27]:
```suggestion
  * Marco will proceed to merge LTS group to release and close the issue.
```

File: doc/meetings/2024-09-19.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+* Creating a branch rule for non LTS staging branches [#1004](https://github.com/nodejs/Release/issues/1004)
+Consensus is that only releasers and backporters should be able to push on staging branches, waiting for a volunteer to update the rule.
```
Comment (richardlau) [inline doc/meetings/2024-09-19.md:29]:
```suggestion
  * Consensus is that only releasers and backporters should be able to push on staging branches, waiting for a volunteer to update the rule.
```
```suggestion
Consensus is that only releasers and backporters should be able to push on staging branches, waiting for a volunteer to update the rule.
```

File: doc/meetings/2024-09-19.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+* Release plan - v22.x Current [#1001](https://github.com/nodejs/Release/issues/1001)
+Rafael volunteering for next 22 release in October.
+v22 will be promoted to LTS the 29th of October.
+v23 needs to be released, looking for volunteers.
```
Comment (richardlau) [inline doc/meetings/2024-09-19.md:33]:
```suggestion
  * Rafael volunteering for next 22 release in October.
  * v22 will be promoted to LTS the 29th of October.
  * v23 needs to be released, looking for volunteers.
```

File: doc/meetings/2024-09-19.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+* Onboarding @aduh95 as a releaser [#999](https://github.com/nodejs/Release/issues/999)
+Antoine ready to be onboarded.
+Richard to reach out re. gpg and ssh keys and complete onboarding steps.
```
Comment (richardlau) [inline doc/meetings/2024-09-19.md:36]:
```suggestion
  * Antoine ready to be onboarded.
  * Richard to reach out re. gpg and ssh keys and complete onboarding steps.
```

File: doc/meetings/2024-09-19.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+* Release plan - v20.x Active LTS [#855](https://github.com/nodejs/Release/issues/855)
+Rafael: that we need more volunteers. Should re-evaluate the current list of releasers, ask to inactive to offboard and try to get more volunteers.
+Richard: collab summit would be a good place to discuss about this
+Marco: we should have a session in the next collab summit
+Last release before v20 goes to maintenance, Marco volunteers for the next v20 in October.
```
Comment (richardlau) [inline doc/meetings/2024-09-19.md:41]:
```suggestion
  * Rafael: that we need more volunteers. Should re-evaluate the current list of releasers, ask to inactive to offboard and try to get more volunteers.
  * Richard: collab summit would be a good place to discuss about this
  * Marco: we should have a session in the next collab summit
  * Last release before v20 goes to maintenance, Marco volunteers for the next v20 in October.
```

File: doc/meetings/2024-09-19.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+* Release plan - v18.x Maintenance LTS [#737](https://github.com/nodejs/Release/issues/737)
+Possibly one release before the end of the Year (maybe security release)
+Some issues with OpenSSL updates, should be released on v22 and backported.
```
Comment (richardlau) [inline doc/meetings/2024-09-19.md:44]:
```suggestion
  * Possibly one release before the end of the Year (maybe security release)
  * Some issues with OpenSSL updates, should be released on v22 and backported.
```

File: doc/meetings/2024-09-19.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+### nodejs/citgm
+
+* Node.js latest CITGM results [#1060](https://github.com/nodejs/citgm/issues/1060)
+ Rafael has fixed some issues in CITGM
```
Comment (richardlau) [inline doc/meetings/2024-09-19.md:49]:
```suggestion
  * Rafael has fixed some issues in CITGM
```

**Review by richardlau — APPROVED** [review]
Formatting suggestions to ease reading in rendered markdown, but otherwise LGTM.

**Review by RafaelGSS — APPROVED** [review]

**Pull request merged**